hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/RootedOzoneFileSystem.java

```diff
@@ -142,12 +149,27 @@ public boolean recoverLease(final Path f) throws IOException {
       }
       throw e;
     }
-    // TODO: query DN to get the final block length
+
     OmKeyInfo keyInfo = infoList.get(0);
+    // finalize the final block and get block length
```

Comment:
Looks like we can extract this method so it can be reused by OzoneFileSystem.recoverLease()